util/OptionsDB.cpp

```diff
@@ -431,6 +431,24 @@ void OptionsDB::SetFromCommandLine(const std::vector<std::string>& args) {
     }
 }
 
+void OptionsDB::SetFromFile(const boost::filesystem::path& file_path,
+                            const std::string& version) {
```

Comment:
I prefer to put the `{` on a new line after a multi-line if expression or function header in order to make it clear at a glance where the actual code starts. Often it's unclear due to both being indented, and if the `{` is hidden off to the right somewhere, it isn't visible enough.

If it's a single-line function, then the `{` on the same line is fine, as I find it clearer what is code and what is definition or if test.